media-gfx/librecad/files/librecad-2.1.3-CVE-2018-19105.patch

```diff
@@ -0,0 +1,92 @@
+From: Markus Koschany <apo@debian.org>
```

Comment:
Please add a link to where you got it from in Debian too and also any relevant Gentoo bugs.

Reply:
fixed